app/utils/constants.js

```diff
@@ -1,14 +1,10 @@
-// COUNTRY CODES
-exports.DOMAIN_ARGENTINA = 'com.ar';
-exports.DOMAIN_COLOMBIA = 'co';
-exports.DOMAIN_CHILE = 'cl';
-exports.DOMAIN_MEXICO = 'com.mx';
+const { availableDomains } = require('../../config').common;
 
 // DOMAINS
-exports.DOMAIN_WOLOX_AR = `wolox.${exports.DOMAIN_ARGENTINA}`;
-exports.DOMAIN_WOLOX_CO = `wolox.${exports.DOMAIN_COLOMBIA}`;
-exports.DOMAIN_WOLOX_CL = `wolox.${exports.DOMAIN_CHILE}`;
-exports.DOMAIN_WOLOX_MX = `wolox.${exports.DOMAIN_MEXICO}`;
+exports.DOMAIN_WOLOX_AR = 'wolox.com.ar';
+exports.DOMAIN_WOLOX_CO = 'wolox.co';
+exports.DOMAIN_WOLOX_CL = 'wolox.cl';
+exports.DOMAIN_WOLOX_MX = 'wolox.com.mx';
 
 // EMAILS
 exports.EMAIL_WOLOX_AR = `@${exports.DOMAIN_WOLOX_AR}`;
```

Comment:
What is the use for these constants?

Reply:
These constants are currently used in email's schema validations.